css-containment/Overview.bs

```diff
@@ -197,8 +197,12 @@ Paint Containment</h3>
 
 	Giving an element <dfn export>paint containment</dfn> has the following effects:
 
-	1. The contents of the element must be clipped to the element's content box.
-		This includes both the paint of the descendants and their geometry (similar to overflow).
+	1. As specified in [[!CSS-OVERFLOW-3]],
+		the computed value the 'overflow-x' and 'overflow-y' is changed to ''overflow/clip''
+		if it would otherwise have been ''overflow/visible''.
```

Comment:
I prefer "If the computed value of overflow-x or overflow-y would otherwise have been 'visible', it must instead compute to 'clip'. [[!CSS-OVERFLOW-3]]"